Tutorial/app/src/main/java/ably/io/tutorial/ExampleActivity.java

```diff
@@ -0,0 +1,49 @@
+package ably.io.tutorial;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.widget.Toast;
+
+import io.ably.lib.realtime.AblyRealtime;
+import io.ably.lib.realtime.ConnectionState;
+import io.ably.lib.realtime.ConnectionStateListener;
+import io.ably.lib.types.AblyException;
+import io.ably.lib.types.ClientOptions;
+
+public class ExampleActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_example);
+        try {
+            initAbly();
+        } catch (AblyException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /* Add AblyException to method signature as AblyRest constructor can throw one */
+    private void initAbly() throws AblyException {
+        ClientOptions clientOptions = new ClientOptions();
+
+		/* Set auth URL where your local server is running, 10.0.2.2 address is needed for emulator to see localhost on your computer */
+        clientOptions.authUrl = "http://10.0.2.2:3000/auth";
```

Comment:
Please see https://github.com/ably/tutorials/pull/25/commits/3ae450dc9d729be2243a735a3c48bb21486e9930#r86729107, the same things applies